refresh_token/controller.py

```diff
@@ -32,6 +33,13 @@ def GET_refresh_token(self, *args, **kwargs):  # pylint: disable=unused-argument
         except NotFound:
             account = register(username, uuid4().hex, '127.0.0.1')
 
+        # this does the same thing that Subreddit.subscribe_default()
+        # should be doing, but doesn't for reasons we didn't want to investigate
+        if not account.has_subscribed:
```

Comment:
What if we add a new default subreddit? Is the plan to do this step manually in the shell?